lms/templates/navigation.html

```diff
@@ -180,6 +159,34 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
       </%block>
     </ol>
     % endif
+    % if static.show_language_selector():
```

Comment:
@mattdrayer header.html and navigation.html are two different files, one is of LMS while other is related to Studio but both are using the common static content file.